packages/web-react/src/components/ScrollView/constants.ts

```diff
@@ -2,6 +2,7 @@ import { Direction } from '../../constants';
 
 export const DEBOUNCE_DELAY = 50;
 export const EDGE_DETECTION_INACCURACY_PX = 1;
+export const SCROLL_CANCEL_DELAY = 10;
```

Comment:
question: What is the unit of this number? Seconds? I think there should be a comment. And the same for the `DEBOUNCE_DELAY`.